File: day four/python/afterbreak/Task5.py
total = 0
average = 0
for number in range(1,11,1):

    number = int(input("Enter number"))

    if(number % 2 == 0):
        total = number + total
   #average = total / 10

#rint("the average is:", average)

print("the total is:", total)

File: day four/java/after break /Task5.java
import java.util.Scanner;
public class Task5 {

public static void main(String[] args){

Scanner input  = new Scanner(System.in);

    int sum = 0;
  
    

    for (int count = 1; count <= 10; count++){

        System.out.print("Enter number");
            int number = input.nextInt();
       
            if(number % 2 == 0){
              sum = sum + number;
            
            }
       
    }
System.out.println("the sum is:" + sum);

}
}
